### ISSUE TITLE
ODBC Pagination

### ISSUE THREAD

**Issue by sarahkemp** [cooperl22/laravel-db2#7]
First, thank you for writing this package! Got me up and running with Laravel much quicker than I expected. The issue I'm seeing is with pagination, it pulls the first page fine, but subsequent pages throw an error: 
```
SQLSTATE[42000]: Syntax error or access violation: 0 [IBM][System i Access ODBC Driver][DB2 for i5/OS]SQL0104 - Token , was not valid. Valid tokens: FROM INTO. (SQLPrepare[0] at /build/php5-8gpaOy/php5-5.5.9+dfsg/ext/pdo_odbc/odbc_driver.c:206) (SQL: select * from (select *, row_number() over (order by 1) as row_num from SHPLOG where SLSHDT = 20151030) as temp_table where row_num between 21 and 40)
```

Trying the query myself (outside of Laravel), I can only get it to execute successfully if the table name qualifies the * selector, e.g.:

```
select * from (select SHPLOG.*, row_number() over (order by 1) as row_num from SHPLOG where SLSHDT = 20151030) as temp_table where row_num between 21 and 40
```

I'm new to Laravel, do you have any suggestions to get my pagination working? Would I be heading in the right direction to try to customize this package to prefix the selector with the table name in queries like this? 

Running the ODBC Access driver on Linux, accessing data on a V7R1 machine.

**Comment by makr8100** [comment]
I can identify the issue with the DB2 syntax, it's exactly what you've found.  DB2 requires table names to be qualified to select * if you're going to add fields to the select clause.  Try aliasing the table as the query builder should handle this properly with the alias included.

**Comment by sarahkemp** [comment]
I really appreciate the response makr8100, I tried aliasing the table in my Model:
```
protected $table = 'SHPLOG as SHPLOG';
```
And it does change my query to:

```
select * from (select *, row_number() over (order by 1) as row_num from SHPLOG as SHPLOG where SLSHDT = 20151030) as temp_table where row_num between 21 and 40
```

But as you can see it still doesn't qualify the select. Is there another method of aliasing the table that you are referring to?

Thanks again.

**Comment by cooperl22** [comment]
Fixed by 2.4
